compiler-rt/lib/builtins/aarch64/sme-abi-vg.c

```diff
@@ -28,22 +28,3 @@ __attribute__((constructor(90))) static void get_aarch64_cpu_features(void) {
 
   __init_cpu_features();
 }
-
-__attribute__((target("sve"))) long
-__arm_get_current_vg(void) __arm_streaming_compatible {
-  struct SME_STATE State = __arm_sme_state();
```

Comment:
Can the definition of this struct and the prototype of `__arm_sme_state` be removed as well?

Reply:
They can indeed :)